x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/_field_list_item.scss

```diff
@@ -24,13 +25,20 @@
   }
 }
 
-.mappingsEditor__fieldsListItem__field--selected {
+.mappingsEditor__fieldsListItem__field--highlighted {
   background-color: $euiColorLightestShade;
   &:hover {
     background-color: $euiColorLightestShade;
   }
 }
 
+.mappingsEditor__fieldsListItem__field--selected {
+  background-color: $euiColorLightShade;
```

Comment:
Can we change this to `$euiTableSelectedColor` to align with the table?

![image](https://user-images.githubusercontent.com/1238659/72303387-faeedc00-3621-11ea-8d78-544b7abcb00a.png)